.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,19 +1,33 @@
-## Testing
+## ℹ️ Description
```

Comment:
Shouldn't that be already in the ticket? Also, why do we want to make this mandatory?

Reply:
Yes and no, the Ticket describes the task/issue, this PR should describe the implementation.

Reply:
Can you expand on that? I don't understand what would be written about the implementation that is noteworthy and not a trade of.

Reply:
there is another thing: If I understand you correctly, we arent supposed to put any text _before_ the template, correct? _Everyone_ did it since this template is active.

Reply:
> there is another thing: If I understand you correctly, we arent supposed to put any text before the template, correct? Everyone did it since this template is active.

Unfortunately you don't. It's perfectly fine to add things if you like, we just decided to not make this mandatory. It's your PR, if you want to have more strucutre feel free to add as much as you like. But please fill out all the structure we provide as well.

Reply:
I would like to have every block there so it's mandatory, yes. Will I reject PRs because people removed them? No!

Reply:
either its mandatory - then we dismiss PRs that doesnt follow the rules (at least from the team), or it is not, than I dont see any issues with this (optional) description block. 🤷‍♂️